### PR TITLE
Fix initial balances in ledger tests

### PR DESCRIPTION
**Pull request by algorandskiy**
## Summary

Some tests compare data against old released behavior and increasing initial balance breaks hard-coded records

## Test Plan

This is a test

### CODE REVIEW

File: ledger/ledger_test.go [hunk 1]
```diff
@@ -1579,6 +1581,12 @@ func TestListAssetsAndApplications(t *testing.T) {
 
 func TestLedgerMemoryLeak(t *testing.T) {
 	t.Skip() // for manual runs only
+	savedBaseAlgo := baseAlgoPerAccount
+	baseAlgoPerAccount = 10000000000
```
Comment (tsachiherman) [inline ledger/ledger_test.go:1585]: this might cause data races if we run two tests that access this variable concurrently.

Comment (tsachiherman) [reply ledger/ledger_test.go:1585]: passing this parameter in is the "safe" approach.

Comment (algorandskiy) [reply ledger/ledger_test.go:1585]: this test is for manual running only (see `t.Skip()`) so I think it is ok to use a global var for it. Passing as a param is safer, agree, but requires 22 modifications that are not really necessary in all these cases.

Comment (algorandskiy) [reply ledger/ledger_test.go]: done, moved to args.

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**